examples/recipes/multiple-data-sources/queries/run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+host=cube
+port=4000
+loadUrl=cubejs-api/v1/load
+readyzUrl=readyz
+
+cubeDevtoken=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwidGVuYW50IjoiY3ViZURldiIsImlhdCI6MTUxNjIzOTAyMiwiZXhwIjoxNzI0OTk1NTgxfQ.i9n4vjnuL3-Ly9aWTbj9pdZQWZpxrle6KIStmD3huqI
+cubeIncToken=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwidGVuYW50IjoiY3ViZUluYyIsImlhdCI6MTUxNjIzOTAyMiwiZXhwIjoxNzI0OTk1NTgxfQ.JWoqbgonhuGhMj0aTIxa4DuZTy1EksN7CaxyF7zi5qE
+
+query=$(cat query/queries/users.json)
+
+# Wait for the Cube API to become ready
+until curl -s "$host":"$port"/"$readyzUrl" > /dev/null; do
+  sleep 1
+done
+
+# Send the query
+curl "$host":"$port"/"$loadUrl" -H "Authorization: ${cubeIncToken}" -G -s --data-urlencode "query=${query}" -o cubeIncResponse.json
+curl "$host":"$port"/"$loadUrl" -H "Authorization: ${cubeDevtoken}" -G -s --data-urlencode "query=${query}" -o cubeDevResponse.json
+
+echo "Cube Inc last users:"
+jq ".data" cubeIncResponse.json
+
+echo "Cube Dev last users:"
+jq ".data" cubeDevResponse.json
```

Comment:
I actually wonder if we can do some command line magic to decode/parse JWTs and display their payloads here as well.

Reply:
I added [some magic](https://github.com/cube-js/cube.js/pull/3341/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.md&file-filters%5B%5D=.sh&file-filters%5B%5D=.sql#diff-c9758e3c67ef412f2707c988539e10461c2652f3fc664884190dfd120be2a82eR3), but it looks a little scary 😄 

Another and more simpler way is using `jq -R 'split(".") | .[1] | @base64d | fromjson' <<< "<token_here>"` but jq in our image don't support `@base64d` and we can't update it easy because image doesn't have `apt`